Formula/edge-impulse-cli.rb

```diff
@@ -0,0 +1,27 @@
+require "language/node"
+
+class EdgeImpulseCli < Formula
+  desc "Command-line interface tools for Edge Impulse"
+  homepage "https://github.com/edgeimpulse/edge-impulse-cli#readme"
+  url "https://registry.npmjs.org/edge-impulse-cli/-/edge-impulse-cli-1.14.2.tgz"
+  sha256 "ecbd5eb1b5acad313db98cf326349bab6a13f663146f3726bce0f497c30991b8"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+  end
```

Comment:
I don't think this is needed, does it not work if you leave this out?

Reply:
Removed